ui/components/ui/popover/index.scss

```diff
@@ -100,7 +101,7 @@
   &-arrow {
     width: 22px;
     height: 22px;
-    background: white;
+    background: var(--color-background-default);
```

Comment:
What is this arrow and where does it show up? Doesn't look great on storybook?

<img width="1440" alt="Screen Shot 2022-03-15 at 12 55 28 PM" src="https://user-images.githubusercontent.com/8112138/158460656-d4c598bb-3a05-4997-b8b6-f27f636c3784.png">

Reply:
I don't know the intended use of this but it seems to be not used.

Reply:
Ah k let's just updated and leave it 😄